spec/support/headers_helpers.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module Spec
+  module Support
+    module Helpers
+      def rack_versioned_headers
```

Comment:
Get rid of this class, use `Grape::HTTP::Headers::CACHE_CONTROL` and friends.

Reply:
The only issue with this is what to do in the specs that are testing with non `Grape::HTTP::Headers` headers, such as `Grape-Likes-Symbolic` or `X-Access-Token`, as these need to be Rack versioned in the specs. 

I've removed the headers helper though and put these inline, so they are defined in whatever specs needed them